dev_cmds/setup-dev.ts

```diff
@@ -4,38 +4,32 @@ import { devcmdPackageDir, devPath, singlePackageJsonExampleDir } from "./utils/
 import fs from "fs-extra";
 
 async function main() {
-
-  if(await fs.pathExists(devPath)) {
-    process.stderr.write('A development folder already exists!\n')
-    process.exit(1)
+  if (await fs.pathExists(devPath)) {
+    throw new Error("A development folder already exists!\n");
```

Comment:
1) it would be great to print the concrete path that's being tried
2) the `\n` is probably also not necessary